src/App.svelte

```diff
@@ -42,14 +44,19 @@
       <Matches />
     </PrivateRoute>
 
-  <PrivateRoute path="new-listing" let:location>
-    <NewListing />
-  </PrivateRoute>
+    <PrivateRoute path="new-listing" let:location>
+      <NewListing />
+    </PrivateRoute>
 
-  <PrivateRoute path="messages" let:location>
-    <Messages />
-  </PrivateRoute>
+    <PrivateRoute path="messages" let:location>
+      <Messages />
+    </PrivateRoute>
 
+    <PrivateRoute path="developer" let:location>
```

Comment:
Is this a tool for when your building? If so probably don't want to commit it.

Reply:
This is a page for developer tools for use while setting things up for demonstration purposes; so any functions that we want to run quickly and easily in button form, we can place there for easy access out of the view